src/middleware/issueDetails.middleware.js

```diff
@@ -86,8 +86,7 @@ async function fetchEntry (req, res, next) {
 
   // look at issue Entries and get the index of that entry - 1
 
-  const entityNum =
-      Object.values(issuesByEntryNumber)[pageNum - 1][0].entry_number
+  const entityNum = Object.values(issuesByEntryNumber)[pageNum - 1][0].entry_number
```

Comment:
https://planning-data-platform.sentry.io/issues/5930021585/?alert_rule_id=15340009&alert_type=issue&notification_uuid=d5af11ea-3885-4bc6-b9da-bf87026c7d77&project=4507033205800960&referrer=slack

I know this line wasn't really changed in this but it's causing errors to show